src/commands/integration/deploy.ts

```diff
@@ -90,6 +96,10 @@ export async function toContract(
 ): Promise<Contract> {
   startWait('Deploying');
 
+  if (contractSpec.files === undefined) {
+    throw new Error(`You need to upgrade henesis-cli to the latest version`);
```

Comment:
`You need to upgrade henesis-cli to the latest version`
 => `You need to update henesis.yaml schema. Please refer docs or run 'henesis init' again for the new schema`